src/model/utilities/computerPlayer.js

```diff
@@ -0,0 +1,24 @@
+const computerPlayer = (playerToExtend) => {
+  const returnedPlayer = playerToExtend;
```

Comment:
Czym jest 'playerToExtend'? Bo jeżeli to obiekt zwykłego 'playera' rozszeżony o nowe funkcje to może zamiast dodawać go tutaj jako argument, lepiej zrobić import i jako returnedPlayer, zrobić obiekt tego playera? 
```suggestion
import player from './Player';

const computerPlayer = () => {
  const returnedPlayer = player();
```

Reply:
Ogólnie computerPlayer jest takim "lepszym graczem" - może odpowiadać automatycznie na odpowiedzi. Nie importowałam tutaj, bo będę player'a przekazywać do niego z silnika gry.